doc/api/fs.md

```diff
@@ -1524,11 +1524,27 @@ If `options` is a string, then it specifies the encoding. Example:
 ```js
 fs.readFile('/etc/passwd', 'utf8', callback);
 ```
+*Note*: When the path is a directory, the behavior of
+`fs.readFile()` and [`fs.readFileSync()`][] is platform-specific. On macOS,
+Linux, and Windows, an error will be returned. On FreeBSD, the contents
+of the files in the directory will be returned.
```

Comment:
@s0m3on3 Have look at the options parameter in the [doc](https://nodejs.org/docs/latest/api/fs.html#fs_fs_readfile_file_options_callback), you can pass the encoding format as "utf-8" to get the contents in a readable format. 

> If no encoding is specified, then the raw buffer is returned.